tracpro/polls/models.py

```diff
@@ -472,7 +472,21 @@ def get_responses(self, region=None, include_subregions=True,
                 responses = responses.filter(contact__region=region)
         if not include_empty:
             responses = responses.exclude(status=Response.STATUS_EMPTY)
-        return responses.select_related('contact')
+
+        responses = responses.select_related('contact')
+
+        # Remove duplicate resopnse from the same user on the same day
+        response_pk_list, response_tuple_set = [], set()
```

Comment:
Initializing several variables in one line this way is valid Python, but my feeling is that readability suffers, and it's generally better to put each one on its own line.

Which is not to say this kind of thing is never appropriate. It's great for breaking out values from a tuple or list into separate variables, or atomically swapping values.